.circleci/install-packr.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+RELEASE_URL="https://api.github.com/repos/gobuffalo/packr/releases/latest"
```

Comment:
I wonder if we might want to lock this down to a particular version so the build will be deterministic on CI. I think the fact that we log out the version number used (it's in the curl output) is good enough for now, cause we can always go back and look at the most recent passing build if something starts failing.

Reply:
@royvandewater That's a great point, after talking with @eric-hu we came up with fab85b2f5d36b0eae8431f3dc802d0a435d7ce5d

Reply:
Sweet!